### DISCUSSION TITLE
Missing a lot of options inmenu.

### DISCUSSION THREAD

**Discussion by yaz72by** [Courseplay/Courseplay_FS22#1729] [Q&A] [answered]
I am missing a lot of options in the menus in vehicle options like speeds for equipment in the field. Multitool settings is not showing up i can not set any speeds through cp and there are some other ones as well but can not think of what they were right now.. Very strange as i watch some yt videos and they all have all these settings available..not sure if it is a mod conflict of some sort.. I have deactivated vehicle ext.  I am using 7.1 also.. I am going to keep trying different things to see if i can figure this out.

**Comment by Tensuko** [comment, marked as answer]
Ingame Help menu got the answer for you -> expert mode!

**Comment by yaz72by** [comment]
Thanks.
